src/scatterLogo.ts

```diff
@@ -1,10 +1,15 @@
-export const scatterLogo: string = `data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAAgCAYAAAFoexhnAAAAAXNSR0IA
-  rs4c6QAAAplJREFUSA2VlE+IjWEUxr97zbi3IRYSIUlZWEpZShLNbhaShSw0ESuFhYWVjYVZU6ipmSQLNkjk3ySRPwtpIotZuEURxZQZjfn8nnPf
-  8973u/e7jXvq+c45z3nOee/7vt/9sgzL83ynvIL9wY8b0f2B9EuFx3YkDTXmURsTBZgkTQXJlCeqxBYnnagmxHGEP5VXvAoxQXxNQweAzcHNF36H
-  Cj5nlRJaan1hRn8F83lq+wSmI6EAohEJkkPAVop95DUUM1HVCs6ZyDvgq6wX9wv/V5ytkYisn3yDOJkI7cFug3ybCPwV3LC2IBNnRmEpcLvgvHs7
-  DRqmra3Z+S6oP7qo1CO6IaEX7RjJNVE7cZslsFWcKHgaLoKv4D5o/XCpIC6DDosTqLwO1UknydcFbqsmHAyJnVciWpKKLPaie8gfKlhuijwf8aI
-  8nC9/RrkdAd5emtipSpaNcbZnLQqT5pQQrwCPwWor+gNiBMjeOlfqEUyYrPkYKBWJpL4nEe7uKgzi+SCuLySU7pKLut90lm10kXm6boE3YBf4DG
-  Tr20W3m3x8nioIuiXIh8DL2FYePIHuPDLINaCs+Tn8STAI9gG/f8L8D2gOI9CH8DpIbZSkc7WwBWrfEvFeXVgVfEjIF8T93bYsnvrRRD9HXBd5M
-  yGfLjCgjvZVoteQTdZDMJsUhtNB8Pru6m0+Br6D1HQUrT83yd2keqdt0KOkpvAXOJFqYkxhLfgN3CYJlkVBLwGNNfDAJwWvN3OwlzlRS2MfOB8G
-  pe49yZYo7CWgcTm4mk4j1sXs6GVOQUvz6baBhwuCXhIGjSfDnrX3Fr4RCPXOaPXFqZBc34uhhHuYxMUQ8UrwP3ak2FmSMWUROADugSkwAxpgDGw
-  uaYnUP9ivtkVEndh2AAAAAElFTkSuQmCC`
+// tslint:disable:max-line-length
+
+// SVG for scatter logo
+
+// <svg width="32px" height="34px" viewBox="13 20 26 5" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Can we remove this commented out code?

Reply:
I left this so anyone wanting to alter it could easily have the source used in the data url without having to reconstruct it.